docs/platforms/dotnet/common/configuration/msbuild/index.mdx

```diff
@@ -0,0 +1,7 @@
+---
+title: MSBuild Setup
```

Comment:
Feedback about the page structure appreciated.

Reply:
I like the split personally.

Reply:
In general I prefer 1 long page, as it's easier to search within the page itself, and documentation is often overwhelming to folks when they jump back and forth between pages.

That said, it seems here one section requires auth (CLI part) while the other one doesn't. The split here makes that distinction more clear. That said, if I'm new to Sentry, I don't know what Sentry CLI is. So the description on front matter should have some hints (e.g: debug file management). 
I'll leave a suggestion